install/helm/agones/templates/controller.yaml

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 {{- $featureGates := include "agones.featureGates" . | fromYaml }}
+{{- $controllerReplica := include "agones.controller.replica" . }}
```

Comment:
nit: I would call this `controllerReplicas` (plural), or just `replicas`. You're in `controller.yaml`, the `controller` prefix is kind of redundant.

Reply:
Done, changed to just `replicas`